internal/controllers/reports.go

```diff
@@ -13,7 +14,35 @@ type ReportsServer struct {
 	store datastore.ReportStore
 }
 
-func (r *ReportsServer) GetMemberCounts(w http.ResponseWriter, req *http.Request) {
+func (r *ReportsServer) GetAccessStatsChart(w http.ResponseWriter, req *http.Request) {
+	resourceName := req.URL.Query().Get("resourceName")
+	day := req.URL.Query().Get("day")
```

Comment:
We don't use this in the UI right? Why have it?

Reply:
for the future!

Reply:
actually, we do use it https://github.com/HackRVA/memberdashboard/blob/71274d4c785311c7dcd60a6646495364af353ac7/web/src/report/services/report.service.ts#L27

Reply:
that's only for the resourceName. Not for `day`. Future is unknown. Unless it's immediate follow up I would prefer we removed unused code whenever.